app/api/v1/endpoints/unified_chat.py

```
@@ -0,0 +1,710 @@
+                            session_id=session_id,
+                            interface=InterfaceType.WEB_CHAT,
+                            conversation_mode=conversation_mode,
+                            stream=True
+                        ):
+                            await websocket.send_text(json.dumps({
+                                "type": "chat_response",
+                                "session_id": session_id,
+                                "chunk": chunk,
+                                "timestamp": datetime.utcnow().isoformat()
+                            }))
+                    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)